File: Backend/CourseAdminSystem.Model/Entities/Recipe.cs
namespace CourseAdminSystem.Model.Entities;

public class Recipe
{
    public Recipe(int recipeid){
        RecipeId = recipeid; 
    }

    public int RecipeId {get; set; }
    public string RecipeName {get; set;}
    public string RecipeInstruct {get; set;}
    public string RecipeIngredients {get; set;}
    public string RecipeStory {get; set;}
    public string RecipeWord {get; set;}
     public string? RecipePicture {get; set;}
     public User user {get; set;}

}

File: CourseAdminSystemAngular/src/app/model/recipe.ts
import { User } from "./user";

export interface Recipe {
recipeWord: string;
    recipeId: number;
    recipeName: string; 
    recipeStory: string; 
    recipeIngredients: string; 
    recipeInstruct: string; 
    recipePicture: string
    user: User;
    
}





